testdata/picard/sam/FilterSamReads/filtersamreads01.js

```diff
@@ -0,0 +1,2 @@
+/** reads mapped and having start%2 = 0 */
```

Comment:
could you give these filters slightly more descriptive names?

maybe `filterOddStarts.js` and `filterReadsWithout5primeSoftClip`